ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBSynthesizer/Run.hs

```diff
@@ -114,9 +113,16 @@ synthesize DBSynthesizerConfig{confOptions, confShelleyGenesis, confDbDir} (Some
         let
             epochSize   = sgEpochLength confShelleyGenesis
             chunkInfo   = Node.nodeImmutableDbChunkInfo (configStorage pInfoConfig)
-            dbArgs      = Node.mkChainDbArgs
-                registry InFuture.dontCheck pInfoConfig pInfoInitLedger chunkInfo $
-                    ChainDB.defaultArgs (Node.stdMkChainDbHasFS confDbDir)
+            dbArgs      =
+             ChainDB.completeChainDbArgs
+              registry
+              InFuture.dontCheck
+              pInfoConfig
+              pInfoInitLedger
+              chunkInfo
+              (const True)
```

Comment:
Are you sure this is the right value?

Reply:
It was using the default args, these is what those args would set.